docs/references/database/list-collections.md

```diff
@@ -1 +1 @@
-Get a list of all the user collections. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's collections. [Learn more about different API modes](/docs/admin).
\ No newline at end of file
+Get a list of all the user's collections. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's collections. [Learn more about different API modes](/docs/admin).
```

Comment:
probably list of all the collections in the `database` with provided `databaseId` or similar?

Reply:
Also query param is only for listDocuments, not sure how we missed it before.

Reply:
Not only this, this also doesn't work in session based SDKs